pyaedt/edb_core/components.py

```diff
@@ -951,7 +952,8 @@ def _create_terminal(self, pin):
         cmp_name = pin.GetComponent().GetName()
         net_name = pin.GetNet().GetName()
         pin_name = pin.GetName()
-        term_name = "{}.{}.{}".format(cmp_name, pin_name, net_name)
+        if term_name is None:
+            term_name = "{}.{}.{}".format(cmp_name, pin_name, net_name)
         for term in list(self._pedb.active_layout.Terminals):
             if term.GetName() == term_name:
                 return term
```

Comment:
@svandenb-dev you reuse the method from PadstackInstance

edb.components["U1"].pins["A1"].get_terminal(name=None, create_new_terminal=True)



Then reuse edb.create_port()